src/server/routes/router.js

```diff
@@ -1,14 +1,15 @@
 const express = require('express')
```

Comment:
Please ensure that you can't connect via WS to the LokIM server while being unauthenticated.